qemu/tests/cfg/numa_memdev_options.cfg

```diff
@@ -0,0 +1,49 @@
+- numa_memdev_options:
+    type = numa_memdev_options
+    kill_vm_on_error = yes
+    numa_hardware_cmd = "numactl --hardware"
+    mem_fixed = 1536
+    mem_devs = "mem0 mem1"
+    backend_mem = memory-backend-ram
+    use_mem_mem0 = no
+    use_mem_mem1 = no
+    host-nodes_mem0 = 0
+    host-nodes_mem1 =1
+    size_mem0 = 512M
+    size_mem1 = 1024M
+    prealloc_mem0 = yes
+    prealloc_mem1 = yes
+    guest_numa_nodes = "node0 node1"
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    variants:
+        - policy_default:
+            policy_mem = default
+            del host-nodes_mem0
+            del host-nodes_mem1
+        - policy_bind:
+            policy_mem = bind
+        - policy_interleave:
+            policy_mem = interleave
+        - policy_preferred:
+            policy_mem = preferred
+    variants:
+        - numa_policy:
+        - numa_host-nodes:
+            no policy_default
```

Comment:
What's the difference between 'numa_policy' and 'numa_host-nodes' besides 'no policy_default' for the latter one?

Reply:
Yes, you have caught the difference. Thanks!

Reply:
Then there should be some duplicate cases, right ?  Such as 'numa_policy.policy_bind'  and 'numa_host-nodes.policy_bind', they are exactly same. Better to remove one to reduce the duplicate work. Thanks.

Reply:
I check "policy" and "host-nodes" attributes in check_query_memdev() and check_memory_in_procfs(). Thus when running case "numa_policy.policy_bind", it will check both "policy" and "host-nodes" though it focuses on "policy" attribute, case "numa_host-nodes.policy_bind" is no difference. The reason why I separate them is to make a one-to-one match with polarion cases ("NUMA memdev option policy test" and "NUMA memdev option host node test"). Of course there are duplicate cases, IMHO, it's better to merge that two polarion cases and then follow your suggestion mentioned above. What's your opinion? @huangyum @jingzhao84

Reply:
@huangyum @ybduan From my opinion, it's much more clearly when the tester executed although there have 3 duplicate test cases. we can separate different combination when executed.

Reply:
@jingzhao84 I would prefer to run all the cases at one time unless for special need.  Discussed with @ybduan , he agrees to remove the duplicate cases too. Thanks!

Reply:
Modified, Thanks!